frontend/apps/picasso/defi/polkadot/pallets/Balances/karura.ts

```diff
@@ -13,13 +13,19 @@ export async function subscribeKaruraBalance(
 ): Promise<() => void> {
   let unsub: UnsubscribePromise = new Promise(() => {});
   try {
-    if (!tokenMetadata.karuraId) throw new Error("Unsupported assets");
+    if (!tokenMetadata.chainId.karura) {
+      return new Promise(() => {});
```

Comment:
shouldn't an error be thrown in this case?

Reply:
No, error wouldn't serve any purpose here and it disrupts the flow. We can just ignore with an empty promise.